<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1314</width>
    <height>1021</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>391</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>接受消息打印窗口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="PtxtRecMess">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="backgroundVisible">
        <bool>false</bool>
       </property>
       <property name="centerOnScroll">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="actionSend">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>871</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::DashLine</enum>
        </property>
        <property name="rowCount">
         <number>20</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>25</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1314</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="FILES">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen_Files"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="EDIT">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="portSwitchAction"/>
    <addaction name="portRunAction"/>
    <addaction name="actionClear_Buffer"/>
    <addaction name="actionPort_Settings"/>
   </widget>
   <widget class="QMenu" name="VIEWS">
    <property name="title">
     <string>Views</string>
    </property>
    <addaction name="actionTool_Bar"/>
    <addaction name="actionStatus_Bar"/>
    <addaction name="actionStays_on_Top"/>
   </widget>
   <widget class="QMenu" name="TOOLS">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionValue_Display"/>
    <addaction name="actionVideo_Box"/>
    <addaction name="separator"/>
    <addaction name="actionSerial_Port_Test_Tool"/>
   </widget>
   <widget class="QMenu" name="HELP">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="FILES"/>
   <addaction name="EDIT"/>
   <addaction name="VIEWS"/>
   <addaction name="TOOLS"/>
   <addaction name="HELP"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPort_Settings"/>
   <addaction name="separator"/>
   <addaction name="portSwitchAction"/>
   <addaction name="portRunAction"/>
   <addaction name="actionClear_Buffer"/>
   <addaction name="actionOptions"/>
  </widget>
  <action name="actionOpen_Files">
   <property name="icon">
    <iconset resource="../resource/rec.qrc">
     <normaloff>:/serialTools/images/open.ico</normaloff>:/serialTools/images/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resource/rec.qrc">
     <normaloff>:/serialTools/images/save.ico</normaloff>:/serialTools/images/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../resource/rec.qrc">
     <normaloff>:/serialTools/images/close.ico</normaloff>:/serialTools/images/close.ico</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="portSwitchAction">
   <property name="icon">
    <iconset resource="../resource/rec.qrc">
     <normaloff>:/serialTools/images/connect.ico</normaloff>:/serialTools/images/connect.ico</iconset>
   </property>
   <property name="text">
    <string>Open Ports</string>
   </property>
  </action>
  <action name="portRunAction">
   <property name="icon">
    <iconset resource="../resource/rec.qrc">
     <normaloff>:/serialTools/images/start.ico</normaloff>:/serialTools/images/start.ico</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionClear_Buffer">
   <property name="icon">
    <iconset resource="../resource/rec.qrc">
     <normaloff>:/serialTools/images/clear.ico</normaloff>:/serialTools/images/clear.ico</iconset>
   </property>
   <property name="text">
    <string>Clear Buffer</string>
   </property>
  </action>
  <action name="actionPort_Settings">
   <property name="icon">
    <iconset resource="../resource/rec.qrc">
     <normaloff>:/serialTools/images/port config.ico</normaloff>:/serialTools/images/port config.ico</iconset>
   </property>
   <property name="text">
    <string>Port Settings</string>
   </property>
  </action>
  <action name="actionTool_Bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tool Bar</string>
   </property>
  </action>
  <action name="actionStatus_Bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status Bar</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="../resource/rec.qrc">
     <normaloff>:/serialTools/images/config.ico</normaloff>:/serialTools/images/config.ico</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionValue_Display">
   <property name="text">
    <string>Value Display</string>
   </property>
  </action>
  <action name="actionVideo_Box">
   <property name="text">
    <string>Video Box</string>
   </property>
  </action>
  <action name="actionStays_on_Top">
   <property name="icon">
    <iconset resource="../resource/rec.qrc">
     <normaloff>:/serialTools/images/pin_up.ico</normaloff>:/serialTools/images/pin_up.ico</iconset>
   </property>
   <property name="text">
    <string>Stays on Top</string>
   </property>
  </action>
  <action name="actionSerial_Port_Test_Tool">
   <property name="text">
    <string>Serial Port Test Tool</string>
   </property>
  </action>
  <action name="actionactionReset">
   <property name="icon">
    <iconset resource="../resource/rec.qrc">
     <normaloff>:/serialTools/images/reset.ico</normaloff>:/serialTools/images/reset.ico</iconset>
   </property>
   <property name="text">
    <string>actionReset</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource/rec.qrc"/>
 </resources>
 <connections/>
</ui>
